cli.js

```diff
@@ -978,12 +981,71 @@ async function init(packageJson, queries, options) {
     },
   );
 
+  const getFF = async () => {
+    var ff = createFFmpeg({log: false});
+    await ff.load();
+    return ff;
+  };
+
   const encodeQueue = new AsyncQueue(
     'cli:postprocessor:encodeQueue',
     Config.concurrency.encoder,
     async ({track, meta, files}) => {
-      return new Promise((res, rej) =>
-        ffmpeg()
+      return new Promise((res, rej) => {
+        getFF()
+          .then(_ffmpeg => {
+            fetchFile(files.audio.file.name)
+              .then(data => {
+                _ffmpeg.FS('writeFile', 'file.x4a', data);
+                _ffmpeg
+                  .run(
+                    '-i',
+                    'file.x4a',
+                    '-y',
+                    '-acodec',
+                    'aac',
+                    '-b:a',
+                    options.bitrate,
+                    '-ar',
+                    '44100',
+                    '-vn',
+                    '-t',
+                    TimeFormat.fromMs(track.duration, 'hh:mm:ss.sss'),
+                    '-f',
+                    'ipod',
+                    'output.m4a',
```

Comment:
`file.x4a`, `output.m4a`?

Reply:
Never mind, it makes sense now. It's running inside a sandbox. Although, we should reuse the loaded wasm context instead of reloading for every new track. Which means we have to use different file names. Just reuse `files.audio.file.name`.

Reply:
I would do that but i have to make a new instance every time because otherwise, since it's asynchronous, it would throw an error about only being able to run once at a time.